mnefun/_mnefun.py

```diff
@@ -610,20 +610,26 @@ def fetch_raw_files(p, subjects, run_indices):
         raw_dir = op.join(subj_dir, p.raw_dir)
         if not op.isdir(raw_dir):
             os.mkdir(raw_dir)
-        finder_stem = 'find %s ' % p.acq_dir
-        # build remote raw file finder
-        fnames = get_raw_fnames(p, subj, 'raw', True, False, run_indices[si])
+        fnames = get_raw_fnames(p, subj, 'raw', True, False,
+                                run_indices[si])
         assert len(fnames) > 0
-        finder = (finder_stem +
+        # build remote raw file finder
+        assert type(p.acq_dir) in (list, string_types)
+        for remote_dir in p.acq_dir:
```

Comment:
this will no longer work for `string_types`. You probably want:
```
use_dirs = [p.acq_dir] if isinstance(p.acq_dir, string_types) else p.acq_dir
for remote_dir in use_dirs:
    ...
```